main.js

```diff
@@ -54,3 +55,29 @@ function dtsPath(path) {
 	const { dir, name, ext } = parsePath(path);
 	return join(dir, `${name}.d${ext}.ts`);
 }
+
+// The same camelCase function from css-loader that is used to provide
+// the class selector names to Gatsby.
+function camelCase(input) {
+  let result = input.trim();
+
+  if (result.length === 0) {
+    return "";
+  }
+
+  if (result.length === 1) {
+    return result.toLowerCase();
```

Comment:
This means the following, right?

CSS

```css
.A {
	color: red;
}
```

Generated TS

```ts
export const a: string;
// NOT `const A`
```

If so, it gives me pause that we modify a valid case.

However, if this logic matches `css-loader`, and that’s the de facto tool used, I’d be happy to make this modification.

Do most tools use this logic out of the box? Vite, NextJS, Webpack 5, and Snowpack come to mind (in that order).